css/cssom-view/scrollLeftTop.html

```diff
@@ -0,0 +1,133 @@
+<!DOCTYPE html>
+<title>CSSOM View - scrollLeft/scrollTop considers writing-mode and css direction</title>
+<meta charset="utf-8">
+<link rel="author" title="Cathie Chen" href="mailto:cathiechen@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/cssom-view/#scrolling-area-origin">
+<link rel="help" href="https://drafts.csswg.org/cssom-view/#scroll-an-element">
```

Comment:
Can you add a meta assert tag to indicate that this test verifies the sign and extreme values of the scroll positions of an element?